_data/sites.json

```
@@ -8181,6 +8181,16 @@
         ]
     },
 
+    {
+        "name": "Wedbio.com",
```

```suggestion
        "name": "Wedbio",
```
Seems like they advertise their name as just `Wedbio`, without the `.com.`